atlp-rwanda/stackup2-barefoot-backend
# validate empty role or email

#### What does this PR do?
- fix a bug while email or role is empty
#### Description of Task to be completed?
- gives an error message when an email is empty
- gives an error message when a role is empty
- write tests for empty email or role 
#### How should this be manually tested?
- After cloning the project repo, run `npm i` inside the cloned repo folder
- Make sure to rename .env.example to .env, 
- In .env file, provide all values for variables under `#Development database` ,  `#General` and
>`SUPER_ADMIN_PASSWORD=`
>`SUPER_ADMIN_EMAIL=`
- Finally, run the following commands respectively:
`git fetch`
`git checkout  bg-admin-assign-role`
`npm i`
`npx sequelize-cli db:create`
`npx sequelize-cli db:migrate`
`npm run admin` to insert super admin
`npm run start`
- To assign role, send the following data in your request body to API at `PATCH http://<your_server_hostname>:<server_port>/api/roles/assign-role` endpoint:
```json
"email":"example@example.com",
"role":"manager"
```
- try to remove the role and see
- remove email also and see  
#### Any background context you want to provide?
you need to sign in as super admin
You need to create an account and use the email of this user.
#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (if appropriate)
N/A
